src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashApiClient.java

```diff
@@ -436,6 +451,91 @@ public String call() throws Exception {
     return response;
   }
 
+  private String putRequest(String path, String status) throws UnsupportedEncodingException {
+    logger.log(Level.FINEST, "PR-PUT-REQUEST:" + path + " with: " + status);
+    HttpClient client = getHttpClient();
+    client.getState().setCredentials(AuthScope.ANY, credentials);
+
+    PutMethod httpput = new PutMethod(path);
+    httpput.setRequestHeader("Connection", "close");
+    httpput.setRequestHeader("X-Atlassian-Token", "no-check"); // xsrf
```

Comment:
remove the comment please https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/